bin/analyze_hoc_activity_functions.rb

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env ruby
```

Comment:
Is this the appropriate directory for these functions? Is this the appropriate filename? Should it be analyze_hoc_activity_helper.rb?